enterprise_access/apps/content_assignments/models.py

```diff
@@ -126,6 +126,7 @@ class Meta:
         max_length=255,
         blank=False,
         null=False,
+        db_index=True,
```

Comment:
i assume this table is currently small in production so indexing wont take long

Reply:
I think it is zero records big.